lib/packs/rails/integrations/rails.rb

```diff
@@ -27,6 +27,10 @@ def create_engines
         def inject_paths
           Packs.all.reject(&:is_gem?).each do |pack|
             Packs::Rails.config.paths.each do |path|
+              # Example case: in rails 6.0, the "config/routes" app path is nil and was not added until
+              # https://github.com/rails/rails/pull/37892/files#diff-a785e41df3f87063a8fcffcac726856a25d8eae6d1f9bca2b36989fe88613f8eR62
+              next unless @app.paths[path]
```

Comment:
I think we should be more explicit and check for the rails version and skip certain paths based off of that. I think with logic, we can potentially skip over paths that we expect to be there but are not for some unknown reason. Wdyt?